inlong-manager/manager-web/sql/apache_inlong_manager.sql

```diff
@@ -16,13 +16,16 @@
  */
 
 SET NAMES utf8mb4;
-SET FOREIGN_KEY_CHECKS = 0;
+SET
```

Comment:
Please not change the code style of SQL file.

Reply:
In addition, the SQL file under the manager-test module also needs to be modified.

Reply:
done